lib/src/components/NxList/NxList.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef, useRef } from 'react';
+import classnames from 'classnames';
+import useMergedRef from '@react-hook/merged-ref';
+import withClass from '../../util/withClass';
+
+import NxLoadingSpinner from '../NxLoadingSpinner/NxLoadingSpinner';
+import NxLoadError from '../NxLoadError/NxLoadError';
+
+import NxListText from './NxListText';
+import NxListSubtext from './NxListSubtext';
+import NxListActions from './NxListActions';
+import NxListButtonItem from './NxListButtonItem';
+import NxListLinkItem from './NxListLinkItem';
+import NxListItem from './NxListItem';
+import { NxListProps, NxListComponent, nxListPropTypes } from './types';
+import useEmptyComponent from '../../util/useEmptyComponent';
+
+
+const NxList = forwardRef<HTMLUListElement, NxListProps>((props: NxListProps, externalRef) => {
+  const {className, children, bulleted, emptyMessage, isLoading = false, error, retryHandler, ...attrs } = props;
+  const classNames = classnames(className, 'nx-list', {'nx-list--bulleted': bulleted});
+  const ulRef = useRef<HTMLUListElement | null>(null);
+  const emptyListRef = useRef<HTMLLIElement>(null);
+  const ref = useMergedRef(ulRef, externalRef);
+  const isEmpty = useEmptyComponent(false, ulRef, emptyListRef);
+
+  if (isEmpty && !isLoading && !error) {
+    if (!emptyMessage) {
+      console.warn('emptyMessage is required when no list items are to be rendered');
+    }
+  }
+
+  const nxListEmpty = (
+    <li ref={emptyListRef} className="nx-list__item nx-list__item--empty">
+      <span className="nx-list__text">
+        {emptyMessage ? emptyMessage : 'This list is empty.'}
+      </span>
+    </li>
+  );
+
+  const nxListLoading = (
+    <li className='nx-list__item'>
```

Comment:
We use double quotes on JSX attrs

Reply:
> We use double quotes on JSX attrs

Fixed.
a47a37756e23cea11ad7e59e78c3502cbed6096d